### PR TITLE
Update delegate.md

### PR DESCRIPTION
**Pull request by pratmit**
Reformatting the release notes

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Comment by ravilach** [comment]
@deepakbolangady not sure if you have someone to review @pratmit's PR? -Ravi

**Comment by pratmit** [comment]
@deepakbolangady This for a new Delegate wizard feature, the proposed changes are very small for the release notes mostly related to formatting.

File: release-notes/delegate.md [hunk 1]
```diff
@@ -21,14 +21,13 @@ Harness NextGen release 78619 includes the following changes for the Harness Del
 ### What's new
 
 - The delegate installation UI experience is now updated with a new installation method; the Terraform Helm Provider. In addition, it has been enhanced for existing methods; Helm Chart, Kubernetes Manifest and Docker. This new experience is more developer friendly, for example it enables cutting and pasting of relevant commands, and more automation friendly where the commands can be run in an automated manner using new values when necessary. Additionally, the following new features are available:
```
Comment (deepakbolangady) [inline release-notes/delegate.md:23]:
```suggestion
- The delegate installation UI experience is now updated with a new installation method: the Terraform Helm provider. Also, the installation experience has been enhanced for the existing methods (Helm chart, Kubernetes manifest, and Docker). This new experience is more developer friendly. For example, it enables cutting and pasting of relevant commands. You can also automate the commands and use new values when necessary. 

Additionally, the following new features are available:
```

File: release-notes/delegate.md [hunk 2]
```diff
@@ -21,14 +21,13 @@ Harness NextGen release 78619 includes the following changes for the Harness Del
...
-  - Custom - Create your own YAML from a [Kubernetes Manifest template](https://github.com/harness/delegate-kubernetes-manifest/blob/main/harness-delegate.yaml) by replacing the placeholders with the values provided in the method.
-  
- Given the need to have a fully qualified YAML, the auto upgrade configuration is set to ON in both the above options. The result is that the delegate version installed will always remain in sync with the version available on the Harness Manager.
- - The **Docker** delegate installation method has now been enhanced to a simple copy and paste of the Docker run command, with the option to modify the pre-populated properties. The auto upgrade is set to OFF for this method with an option to enable it in the command. (DEL-6037)
+  - The **Terraform Helm Provider** method is powered by the open source [Terraform Harness Delegate module](https://registry.terraform.io/modules/harness/harness-delegate/kubernetes/latest) and the open source [Harness Delegate Helm Chart](https://github.com/harness/delegate-helm-chart). The auto upgrade is set to OFF with an option to enable it in the command.
```
Comment (deepakbolangady) [inline release-notes/delegate.md:24]:
```suggestion
  - The **Terraform Helm Provider** method is based on the open source [Terraform Harness Delegate module](https://registry.terraform.io/modules/harness/harness-delegate/kubernetes/latest) and the open source [Harness Delegate Helm chart](https://github.com/harness/delegate-helm-chart). Auto upgrade is set to `OFF` with an option to enable it in the command.
```

File: release-notes/delegate.md [hunk 3]
```diff
@@ -21,14 +21,13 @@ Harness NextGen release 78619 includes the following changes for the Harness Del
...
+  - The updated method for **Helm Chart** is also powered by the same open source [Harness Delegate Helm Chart](https://github.com/harness/delegate-helm-chart) as the Terraform Helm Provider above. The auto upgrade is set to OFF with an option to enable it in the command. You can also download the [default values.yaml](https://github.com/harness/delegate-helm-chart/blob/main/harness-delegate-ng/values.yaml) for the Helm option and edit that to set your own long-lived configuration values.
```
Comment (deepakbolangady) [inline release-notes/delegate.md:25]:
```suggestion
  - The updated method for **Helm Chart** is also based on the same open source [Harness Delegate Helm chart](https://github.com/harness/delegate-helm-chart) as the Terraform Helm provider. Auto upgrade is set to OFF with an option to enable it in the command. You can also download the [default values.yaml](https://github.com/harness/delegate-helm-chart/blob/main/harness-delegate-ng/values.yaml) for the Helm option and edit that to set your own long-lived configuration values.
```

File: release-notes/delegate.md [hunk 4]
```diff
@@ -21,14 +21,13 @@ Harness NextGen release 78619 includes the following changes for the Harness Del
...
+  - The updated flow for **Kubernetes Manifest** has two options for creating a manifest YAML file specific to your Harness account.
```
Comment (deepakbolangady) [inline release-notes/delegate.md:26]:
```suggestion
  - The updated flow for **Kubernetes Manifest** has the following options for creating a manifest YAML file specific to your Harness account. 
```

File: release-notes/delegate.md [hunk 5]
```diff
@@ -21,14 +21,13 @@ Harness NextGen release 78619 includes the following changes for the Harness Del
...
+  - The **Terraform Helm Provider** method is powered by the open source [Terraform Harness Delegate module](https://registry.terraform.io/modules/harness/harness-delegate/kubernetes/latest) and the open source [Harness Delegate Helm Chart](https://github.com/harness/delegate-helm-chart). The auto upgrade is set to OFF with an option to enable it in the command.
+  - The updated method for **Helm Chart** is also powered by the same open source [Harness Delegate Helm Chart](https://github.com/harness/delegate-helm-chart) as the Terraform Helm Provider above. The auto upgrade is set to OFF with an option to enable it in the command. You can also download the [default values.yaml](https://github.com/harness/delegate-helm-chart/blob/main/harness-delegate-ng/values.yaml) for the Helm option and edit that to set your own long-lived configuration values.
+  - The updated flow for **Kubernetes Manifest** has two options for creating a manifest YAML file specific to your Harness account. 
+    - **Basic** - Provides a “Download YAML” option where the downloaded YAML has all the configuration variables pre-populated with values specific to your Harness account.
```
Comment (deepakbolangady) [inline release-notes/delegate.md:27]:
```suggestion
    - **Basic**: Provides a **Download YAML** option. The downloaded YAML has all the configuration variables set to values that are specific to your Harness account. 
```

File: release-notes/delegate.md [hunk 6]
```diff
@@ -21,14 +21,13 @@ Harness NextGen release 78619 includes the following changes for the Harness Del
...
+  - The updated method for **Helm Chart** is also powered by the same open source [Harness Delegate Helm Chart](https://github.com/harness/delegate-helm-chart) as the Terraform Helm Provider above. The auto upgrade is set to OFF with an option to enable it in the command. You can also download the [default values.yaml](https://github.com/harness/delegate-helm-chart/blob/main/harness-delegate-ng/values.yaml) for the Helm option and edit that to set your own long-lived configuration values.
+  - The updated flow for **Kubernetes Manifest** has two options for creating a manifest YAML file specific to your Harness account. 
+    - **Basic** - Provides a “Download YAML” option where the downloaded YAML has all the configuration variables pre-populated with values specific to your Harness account. 
+    - **Custom** - Create your own YAML from a [Kubernetes Manifest template](https://github.com/harness/delegate-kubernetes-manifest/blob/main/harness-delegate.yaml) by replacing the placeholders with the values provided in the method.
```
Comment (deepakbolangady) [inline release-notes/delegate.md:28]:
```suggestion
    - **Custom** - Create your own YAML from a [Kubernetes manifest template](https://github.com/harness/delegate-kubernetes-manifest/blob/main/harness-delegate.yaml) by replacing the placeholders with the values provided in the method.
```

File: release-notes/delegate.md [hunk 7]
```diff
@@ -21,14 +21,13 @@ Harness NextGen release 78619 includes the following changes for the Harness Del
...
+  Given the need to have a fully qualified YAML, the auto upgrade configuration is set to ON in both the above options. The result is that the delegate version installed will always remain in sync with the version available on the Harness Manager.
```
Comment (deepakbolangady) [inline release-notes/delegate.md:29]:
```suggestion
  Given the need to have a fully qualified YAML, the auto upgrade configuration is set to ON in both the above options. Consequently, the delegate version that is installed always remains in sync with the version available on Harness Manager.
```

File: release-notes/delegate.md [hunk 8]
```diff
@@ -21,14 +21,13 @@ Harness NextGen release 78619 includes the following changes for the Harness Del
...
+  - The **Docker** delegate installation method has now been enhanced to a simple copy and paste of the Docker run command, with the option to modify the pre-populated properties. The auto upgrade is set to OFF for this method with an option to enable it in the command. (DEL-6037)
```
Comment (deepakbolangady) [inline release-notes/delegate.md:30]:
```suggestion
  - The **Docker** delegate installation method has now been simplified to a copy-and-paste action on the `docker run` command, with the option to modify the preset values. The auto upgrade is set to OFF for this method, with an option to enable it in the command. (DEL-6037)
```

**Review by deepakbolangady — CHANGES_REQUESTED** [review]
Hi, Prateek. Please get back to me if you have questions. I've used suggestions so you can incorporate this feedback in one click if you wish. Can you please incorporate as required and let me know? I'll approve right after. Thank you.

**Review by deepakbolangady — APPROVED** [review]
LGTM and approved. Thanks, Prateek.

**Comment by deepakbolangady** [comment]
Discussed this failure with Ravi. I learned that the check that's failing is not critical. I have his go-ahead for a force-merge.

**Pull request merged**